src/Chapter1.hs

```diff
@@ -641,7 +671,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | abs n < 10 = abs n
+  | otherwise = firstDigit ((abs n - abs n `mod` 10) `div` 10)
```

Comment:
No need to subtract `abs n nod 10` as it won't affect the `div` result 🙂 

On the other note, you reuse `abs n` a lot in this function. That usually means that you can move it into a separate variable in the `where` clause, and use everywhere 🙂